scheduler/src/cook/mesos/api.clj

```diff
@@ -734,18 +738,15 @@
                 :job.state/waiting "waiting")
         instances (map (fn [instance]
                          (let [hostname (:instance/hostname instance)
-                               executor-states (get-executor-states fid hostname)
-                               url-path (try
-                                          (executor-state->url-path hostname (get executor-states (:instance/executor-id instance)))
-                                          (catch Exception e
-                                            nil))
+                               executor-id (:instance/executor-id instance)
+                               url-path (retrieve-url-path fid hostname executor-id)
```

Comment:
Is this the only place the function is called? Asking since we moved the `catch` to the function proper